docs/source/Setup/Installation.md

```diff
@@ -13,116 +13,114 @@ You can also  [clone Evennia from github](./Installation-Git.md)  or use [docker
 Installing Evennia doesn't make anything visible online. Apart from installation and updating, you can develop your game without any internet connection if you want to.
 ```
 - Evennia requires [Python](https://www.python.org/downloads/) 3.10 or 3.11 (recommended). Any OS that supports Python should work.
-	- _Windows_: In the installer, make sure you select `add python to path`.  If you have multiple versions of Python installed, use `py` command instead of `python` to have Windows automatically use the latest.
+	- _Windows_: In the installer, make sure to select `add python to path`. If you have multiple versions of Python installed, use `py` command instead of `python` to have Windows automatically use the latest.
 - Don't install Evennia as administrator or superuser. 
 - If you run into trouble, see [installation troubleshooting](./Installation-Troubleshooting.md).
 
 ## Install with `pip`
 
 ```{important}
-Using a light-weight [Python virtual environment](./Installation-Git.md#virtualenv) is optional, but _highly recommended_ in order to keep your Evennia installation independent from the system libraries. The virtualenv system is a part of Python and will make your life easier!
+Using a light-weight [Python virtual environment](./Installation-Git.md#virtualenv) is optional, but _highly recommended, to keep your Evennia installation independent from the system libraries. The virtualenv system is a part of Python and will make your life easier!
 ```
 
-Evennia  is managed from the terminal (console/Command Prompt on Windows). Once you have Python, you - after activating your virtualenv if you are using one - install Evennia with
+Evennia is managed from the terminal (console/Command Prompt on Windows). Once you have Python, -- after activating your virtualenv if you are using one -- install Evennia with:
```

Comment:
`--` has no particular meaning in Markdown, it does _not_ turn into a longer typographic dash in html. I don't think that the change hurts either, so I don't request a change; but I'm not convinced there is a particular value with using a double-dash here.

Reply:
Agreed. I wonder if `&mdash;` or `&ndash;` get converted to their respective symbols?

Reply:
&mdash; does get converted in markdown, so—I suggest—it be used for parenthetical asides.

Reply:
I honestly prefer to keep the markdown easier to read rather than adding HTML markup in it just for a slightly longer dash ...